src/helpers/lemmy.ts

```diff
@@ -10,6 +10,21 @@ export const LEMMY_SERVERS =
     ? (window.CUSTOM_LEMMY_SERVERS as string[])
     : ["lemmy.world", "lemmy.ml", "beehaw.org", "sh.itjust.works"];
 
+export const UNPROXIED_LEMMY_SERVERS =
+  "UNPROXIED_LEMMY_SERVERS" in window
+    ? (window.UNPROXIED_LEMMY_SERVERS as string[])
+    : [
+        "lemmy.ml",
+        "beehaw.org",
+        "sh.itjust.works",
+        "lemm.ee",
+        "feddit.de",
+        "midwest.social",
+        "lemmynsfw.com",
+        "lemmy.ca",
+        "lemmy.sdf.org",
```

Comment:
I checked all these instances and they work fine without the proxy.

Reply:
Does lemmy.world require proxy?

Reply:
Yes, it's sending double CORS headers so it breaks.